WooCommerce/Classes/POS/Presentation/Reusable Views/Buttons/POSButtonStyle.swift

```diff
@@ -166,6 +235,20 @@ struct POSButtonStyle_Previews: View {
                 .disabled(true)
         }
     }
+
+    private func loadingPreviewSection(title: String, size: POSButtonSize) -> some View {
+        VStack(alignment: .leading, spacing: 16) {
+            Text(title)
+                .font(.headline)
+
+            Button("Enabled Button") {}
+                .buttonStyle(POSFilledButtonStyle(size: size, isLoading: true))
+
+            Button("Disabled Button") {}
+                .buttonStyle(POSFilledButtonStyle(size: size, isLoading: true))
+                .disabled(true)
```

Comment:
As I understand, the loading state doesn't have a disabled option so both disabled and enabled should look the same, correct?

Reply:
Correct, disabled & enabled states should look the same as in the preview. It's still a button with an action handler, and if we don't want to trigger the action on tap the button has to be disabled separately with `disabled(true)`.